src/lib/analytics/analytics.js

```diff
@@ -301,7 +301,7 @@ const patchLogger = () => {
   // for error logs if they happen frequently only log one
   const debounceFireEvent = debounce(fireEvent, 500, { leading: true })
 
-  logger.error = (...args) => {
+  logger.error = async (...args) => {
```

Comment:
we don't awaiting to the log error anywhere. so this is unnecessary
let's keep it non-async, just will add .finally chain block restoring error message to the Sentry.flush
```suggestion
  logger.error = (...args) => {
```